Application/EdFi.Ods.Features/EdFi.Ods.Features.csproj

```diff
@@ -20,7 +20,7 @@
     <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="7.2.0" />
     <PackageReference Include="Microsoft.Extensions.Caching.Redis" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Caching.SqlServer" Version="6.0.6" />
-    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.16" />
+    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.18" />
```

Comment:
@semalaiappan Security.DataAcess package does not appear to be used in this project

Reply:
@ericstockdale  removed